frontend/components/Landing/Counters.vue

```diff
@@ -1,5 +1,39 @@
 <script setup>
 import { PresentationChartBarIcon, UsersIcon, ChatBubbleLeftIcon } from '@heroicons/vue/24/solid'
+import { onMounted, ref, computed } from 'vue'
+
+const data = ref([])
+
+onMounted(() => {
+   fetch('/api/meetups.json').then((response) => response.json()).then((meetup) => {
+    data.value = meetup
+  })
+})
+
+const countPresentations = computed(() => {
+  let presentations = 0;
+  for(let meetup of data.value) {
+    for(let presentation of meetup.presentations){
+      presentations++;
+    }
+  }
+  return presentations
+})
+
+const countSpeakers = computed(() => {
+  let speakers = []; 
+
+  for(let meetup of data.value) {
+    for(let presentation of meetup.presentations){
+      for(let speaker of presentation.speakers){
+        speakers.push(speaker.name);
+      }
+    }
+  }
+  let uniqueSpeakers = [...new Set(speakers)].length;
```

Comment:
```suggestion
  let uniqueSpeakers = speakers.length;
```

Whhy not?

Reply:
'Set' let me store unique values, if I remove it I will have doubled speakers